tests/chart_tests/test_authsidecar.py

```diff
@@ -54,6 +54,40 @@ def test_authSidecar_alertmanager(self, kube_version):
         assert "NetworkPolicy" == docs[3]["kind"]
         assert [{"port": 8084, "protocol": "TCP"}] == jmespath.search("spec.ingress[*].ports[1]", docs[3])
 
+    def test_authSidecar_houston_with_custom_resources(self, kube_version):
+        """Test custom resources are applied on Houston"""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={
+                "global": {
+                    "authSidecar": {
+                        "enabled": True,
+                        "repository": "someregistry.io/my-custom-image",
+                        "tag": "my-custom-tag",
+                        "resources": {"limits": {"cpu": "500m", "memory": "256Mi"}, "requests": {"cpu": "200m", "memory": "128Mi"}},
+                    }
+                }
+            },
+            show_only=[
+                "charts/astronomer/templates/houston/houston-configmap.yaml",
+            ],
+        )
+
+        common_houston_config_test_cases(docs)
+        prod = yaml.safe_load(docs[0]["data"]["production.yaml"])
+
+        expected_output = {
+            "enabled": True,
+            "repository": "someregistry.io/my-custom-image",
+            "tag": "my-custom-tag",
+            "port": 8084,
+            "pullPolicy": "IfNotPresent",
+            "annotations": {},
+            "resources": {"limits": {"cpu": "500m", "memory": "256Mi"}, "requests": {"cpu": "200m", "memory": "128Mi"}},
```

Comment:
We should make these test values obviously wrong so they are easy to pick out in logs and never accidentally align with what we might have as defaults:

```suggestion
                        "resources": {"limits": {"cpu": "999m", "memory": "888Mi"}, "requests": {"cpu": "777m", "memory": "666Mi"}},
                    }
                }
            },
            show_only=[
                "charts/astronomer/templates/houston/houston-configmap.yaml",
            ],
        )

        common_houston_config_test_cases(docs)
        prod = yaml.safe_load(docs[0]["data"]["production.yaml"])

        expected_output = {
            "enabled": True,
            "repository": "someregistry.io/my-custom-image",
            "tag": "my-custom-tag",
            "port": 8084,
            "pullPolicy": "IfNotPresent",
            "annotations": {},
            "resources": {"limits": {"cpu": "999m", "memory": "888Mi"}, "requests": {"cpu": "777m", "memory": "666Mi"}},
```